indexer/src/test/java/org/pdxfinder/dataexport/DataExportTest.java

```diff
@@ -0,0 +1,282 @@
+        when(dataImportService.findModelBySample(sample)).thenReturn(modelCreation);
+
+        universalDataExporter.setDs(providerGroup);
+        universalDataExporter.initPatientTumorAtCollection();
+
+        List<List<String>> patientTumorAtCollection = universalDataExporter.getPatientTumorSheetData();
+        Assert.assertEquals("p123", patientTumorAtCollection.get(0).get(0));
+    }
+
+    @Test
+    public void TestPdxModelDetailsAndValidations(){
+
+        when( dataImportService.findModelsWithSpecimensAndQAByDS(providerGroup.getAbbreviation())).thenReturn(getModelListForTest());
+
+        universalDataExporter.setDs(providerGroup);
+        universalDataExporter.initPdxModelDetails();
+        universalDataExporter.initPdxModelValidations();
+
+        List<List<String>> pdxModelDetails = universalDataExporter.getPdxModelSheetData();
+        List<List<String>> pdxModelValidations = universalDataExporter.getPdxModelValidationSheetData();
+
+        Assert.assertEquals("m123", pdxModelDetails.get(0).get(0));
+        Assert.assertEquals("hsname", pdxModelDetails.get(0).get(1));
+
+        Assert.assertEquals("m123", pdxModelValidations.get(0).get(0));
+        Assert.assertEquals("technology", pdxModelValidations.get(0).get(1));
+
+    }
+
+    @Test
+    public void TestSharingAndContact(){
+
+        when(dataImportService.findModelsWithSharingAndContactByDS(providerGroup.getAbbreviation())).thenReturn(getModelListForTest());
+
+        universalDataExporter.setDs(providerGroup);
+        universalDataExporter.initSharingAndContact();
+
+        List<List<String>> sharingAndContact = universalDataExporter.getSharingAndContactSheetData();
+
+        Assert.assertEquals("m123", sharingAndContact.get(0).get(0));
+        Assert.assertEquals("Academia", sharingAndContact.get(0).get(1));
+
+    }
+
+    @Test
+    public void TestLoaderRelatedData(){
+
+        universalDataExporter.setDs(providerGroup);
+        universalDataExporter.initLoaderRelatedData();
+
+        List<List<String>> loaderRelatedData = universalDataExporter.getLoaderRelatedDataSheetData();
+
+        Assert.assertEquals("TG", loaderRelatedData.get(0).get(1));
+
+
+    }
+
+    @Test
+    public void TestSamplePlatformDescription(){
+
+        when(dataImportService.findModelXenograftPlatformSampleByDS(providerGroup.getAbbreviation())).thenReturn(getModelListForTest());
+
+        universalDataExporter.setDs(providerGroup);
+        universalDataExporter.initSamplePlatformDescription();
+
+        List<List<String>> samplePlatformDescription = universalDataExporter.getSamplePlatformDescriptionSheetData();
+
+        Assert.assertEquals("s123", samplePlatformDescription.get(0).get(0));
+        Assert.assertEquals("xs123", samplePlatformDescription.get(1).get(0));
+
+    }
+
+    @Test
+    public void TestGenomicData(){
+
+        when(dataImportService.findModelsWithSharingAndContactByDS(providerGroup.getAbbreviation())).thenReturn(getModelListForTest());
```

Comment:
This and following `when()` chains should be formatted to read easier, e.g.
```suggestion
        when(
                dataImportService
                .findModelsWithSharingAndContactByDS(
                        providerGroup.getAbbreviation()
                )
        ).thenReturn(
                getModelListForTest()
        );
```